votes/initiateNewVote/_EDIT_ME.yml

```diff
@@ -24,8 +24,12 @@ headerInstructions: |
 #    voters express their preference for each candidates, no matter how many
 #    there are.
 candidates:
-  - TODO
-  - TODO
+  - Throw when passed to `--env-file` flag, provide no other flag (status quo).
+  - Warn when passed to `--env-file` flag, provide no other flag.
+  - Throw when passed to `--env-file` flag, warn when passed to a new `--env-file-if-exists`.
+  - Throw when passed to `--env-file` flag, do nothing when passed to a new `--env-file-if-exists`.
+  - Throw when passed to `--env-file` flag, unless new `--env-file-when-missing=warn` is also passed.
+  - Warn when passed to `--env-file` flag, unless new `--env-file-when-missing=throw` is also passed.
```

Comment:
I don't think anyone has been advocating for these. These options would mean you can't have a mix of required and optional files to load.

Reply:
> I don't think anyone has been advocating for these.

I have


> These options would mean you can't have a mix of required and optional files to load.

That's correct

Reply:
Sorry I missed that. I feel like a bunch of people would be upset if this is what we land, because the use case they want to support is “I have some env files that every environment needs, plus another env file that only localhost needs”. That was what prompted `--env-file-optional` if I remember correctly.

Reply:
> I feel like a bunch of people would be upset if this is what we land

Yes probably, and that applies to all the candidates BTW – that's why we're having a vote, because we couldn't find a solution that would please everybody.